ios/browser/api/sync/brave_sync_api.mm

```diff
@@ -180,12 +181,14 @@ - (NSString*)getDeviceListJSON {
 
   base::Value device_list_value(base::Value::Type::LIST);
 
-  for (const auto& device : _worker->GetDeviceList()) {
+  for (const auto& device : device_list) {
     auto device_value = base::Value::FromUniquePtrValue(device->ToValue());
     bool is_current_device =
         local_device_info ? local_device_info->guid() == device->guid() : false;
     device_value.SetBoolKey("isCurrentDevice", is_current_device);
     device_value.SetStringKey("guid", device->guid());
+    device_value.SetBoolKey("supportsSelfDelete",
```

Comment:
I can't find where `supportsSelfDelete` is used  neither in branch `ios/sync-delete-devices` nor in https://github.com/brave/brave-ios .  Or this is supposed in upcoming PRs?

Reply:
It would decide whether we allow users to delete remote devices from device list

Reply:
yeah, so this is for future use?